modules/sdk-coin-ada/src/lib/transaction.ts

```diff
@@ -15,9 +15,15 @@ export interface TransactionInput {
   transaction_index: number;
 }
 
+export interface AssetList {
```

Comment:
Do we want this to be called `AssetList` ? Or would `Asset` make more sense?

Reply:
Asset sounds better, will change